oidc-js/src/client.ts

```diff
@@ -39,7 +39,8 @@ import {
     handleSignIn,
     handleSignOut,
     resetOPConfiguration,
-    sendRevokeTokenRequest
+    sendRevokeTokenRequest,
+    isLoggedOut
```

Comment:
isLoggedOut should be alphabetically sorted.

Reply:
Fixed by 86e4c640